src/legacy/components/ReplicateUserFromTemplate.component.js

```diff
@@ -57,7 +57,8 @@ class ReplicateUserFromTemplate extends React.Component {
     };
 
     onUpdateFormStatus = formStatus => {
-        const isValid = !formStatus.asyncValidating && formStatus.valid;
+        const asyncValidating = "asyncValidating" in formStatus && formStatus.asyncValidating;
```

Comment:
We shouldn't use "property" in "object". I forget exactly why Alexis didn't want to use it, maybe for performance. I think instead you should use lodash mapValues and see if the key is there. @SferaDev can chime in if this is actually an ok situation to use property in object and not mapValues. 

Here's an example of code that I corrected from sharing-settings-app when I wanted to originally do `if('visualizations' in payload)`

`function extractExtraDependencies(payload: MetadataPayload): string[] {`
  `  return _(payload)`
       ` .mapValues((value, key) => {`
           ` if (key === "visualizations") {`
               ` return _.flatten(`
                   ` value.map((element: Visualization) =>`
                       ` _.flatMap(element.dataDimensionItems ?? [], (item: DataDimensionItem) => {`
                            `const indicator = item.indicator?.id;`
                            `const programIndicator = item.programIndicator?.id;`
                           ` return _.compact([indicator, programIndicator]);`
                       ` })`

Reply:
Agreed, the ``property in object`` could be refactored to a more functional approach

Reply:
Refactored to remove `in`